tests/unit/scripting.tcl

```diff
@@ -1404,71 +1404,117 @@ start_server {tags {"scripting"}} {
 # Additional eval only tests
 start_server {tags {"scripting"}} {
     test "Consistent eval error reporting" {
+        r config resetstat
         r config set maxmemory 1
         # Script aborted due to Redis state (OOM) should report script execution error with detailed internal error
-        assert_error {ERR Error running script (call to *): @user_script:*: OOM command not allowed when used memory > 'maxmemory'.} {
+        assert_error {OOM command not allowed when used memory > 'maxmemory'*} {
             r eval {return redis.call('set','x','y')} 1 x
         }
+        assert_equal [errorrstat OOM r] {count=1}
```

Comment:
let's improve all these assertions after each test to also check `total_error_replies` and the command stats.

Reply:
Fixed